src/py/_pyodide/_core_docs.py

```diff
@@ -124,6 +124,36 @@ def to_bytes(self) -> bytes:
         an ArrayBuffer view.
         """
 
+    def to_file(self, fd: int):
+        """Writes the entire buffer to a file descriptor.
+
+        Will write the entire contents of the buffer to the current position of
+        the file descriptor.
+
+        Present only if the wrapped Javascript object is an ArrayBuffer or an
+        ArrayBuffer view.
```

Comment:
I think it would be help to have an example (adapted from your unit test) and an equivalent code that makes more copies. For instance,
```suggestion
ArrayBuffer view.

Example
------------
>>> from js import Uint8Array
>>> x = Uint8Array.new(range(10))
>>> with open('file.bin', 'w') as fh:
...    x.to_file(fh.fileno())

which is equivalent to,

>>> with open('file.bin', 'w') as fh:
...    fh.write(x.to_py().tobytes())

but the latter is less efficient and makes more memory copies.
```


(might need to add `# doctest: +SKIP`, or skip it in `conftest.py`) as there is probably no way to make it work from Python for doctests without building this for WASM.